src/FastSerialization/FastSerialization.cs

```diff
@@ -98,14 +98,41 @@ enum StreamReaderAlignment : int
     };
 
     /// <summary>
-    /// These settings apply to use of Serializer and Deserializer specifically.
+    /// Settings used by <code>Serializer</code> and <code>Deserializer</code>.
     /// </summary>
 #if FASTSERIALIZATION_PUBLIC
     public
 #endif
-    sealed class SerializationConfiguration
+    sealed class SerializationSettings
     {
-        public StreamLabelWidth StreamLabelWidth { get; set; } = StreamLabelWidth.EightBytes;
+        internal StreamLabelWidth StreamLabelWidth { get; }
+
+        internal StreamReaderAlignment StreamReaderAlignment { get; }
+
+        public static SerializationSettings Default { get; } = new SerializationSettings(
+            StreamLabelWidth.EightBytes,
+            StreamReaderAlignment.EightBytes);
+
+        public SerializationSettings SetStreamLabelWidth(StreamLabelWidth width)
```

Comment:
Given that this is not setting anything (on current instance), it might be a better idea to use `WithXxx` naming pattern.